forest/src/daemon.rs

```diff
@@ -436,28 +442,52 @@ mod test {
     use fvm_shared::address::Address;
 
     #[async_std::test]
-    async fn import_snapshot_from_file() {
+    async fn import_snapshot_from_file_valid() -> anyhow::Result<()> {
+        anyhow::ensure!(import_snapshot_from_file("test_files/chain4.car")
+            .await
+            .is_ok());
+        Ok(())
```

Comment:
Are there some hidden features of this approach as opposed to a classical assertion without `anyhow::Result` as the return type?
```
async fn import_snapshot_from_file_invalid() {
  assert!(import_snapshot_from_file("test_files/chain4.car").await.is_ok())
}
```

Reply:
The error message will be more descriptive, including the failing expression
`import_snapshot_from_file("test_files/chain4.car")
            .await
            .is_ok()`

Reply:
Alright, sounds good.